protocols/grp/api/grp.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package bio.bgp;
+
+import "net/api/net.proto";
+import "route/api/route.proto";
+
+option go_package = "github.com/bio-routing/bio-rd/protocols/grpcrp/api";
+
+message GRPRoute {
+    string vrf = 1;
```

Comment:
Given the rib in / rib out design: How does that map to VRFs? Are we going to have a RIB in and a RIB out per VRF then?

Reply:
If we think about GRP being kind of a BGP without BGP maybe this is then more a GRMP (gRPC Routing Monitoring Protocol)?

Reply:
GRP is designed to be VRF-aware, so every prefix is send/received with a VRF name provided, so we're basically multiplexing all VRFs over the gRPC connection and the client has to MUX/DEMUX this into the existing VRFs and AFs within Bio.

To GRP vs. GRMP: GRP is designed to be two way, so not only monitoring, hence I'd not add monitoring to the name, as it would be misleading.